src/renderer/coremods/badges/index.tsx

```diff
@@ -105,35 +105,33 @@ export async function start(): Promise<void> {
         return res;
       }
 
+      const badgeTypes = [
+        { type: "booster", component: Badge.Booster },
+        { type: "hunter", component: Badge.BugHunter },
+        { type: "contributor", component: Badge.Contributor },
+        { type: "developer", component: Badge.Developer },
+        { type: "early", component: Badge.EarlyUser },
+        { type: "staff", component: Badge.Staff },
+        { type: "support", component: Badge.Support },
+        { type: "translator", component: Badge.Translator },
+      ];
+
+      Object.entries(badges).forEach(([type, value]) => {
```

Comment:
This can be written much better by looping though `badgeTypes` and checking the API data from that. This will also ensure a consistent order.